org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/XMLLanguageServer.java

```diff
@@ -297,4 +304,14 @@ public void sendNotification(String message, MessageType messageType, Command...
 	public SharedSettings getSharedSettings() {
 		return xmlTextDocumentService.getSharedSettings();
 	}
+
+	@Override
+	public Collection<DOMDocument> allDocuments() {
+		return xmlTextDocumentService.allDocuments().stream().map(m -> m.getModel().getNow(null)).collect(Collectors.toList());

```

Comment:
getNow(null) could return null, I think we should filter this case to avoid having null item in the collection.

Reply:
Fixed